src/routes/public/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Route } from 'react-router';
+import { SignUp, SignIn } from '../../components/registration';
+
+export const PublicRoutes = () : JSX.Element =>
```

Comment:
shouldn't the home / dashboard routes be also present here? Where you display the main interface